sessions/3_visualization.md

```diff
@@ -4,4 +4,82 @@ title: "Hands-on session 3: Designing accessible biomedical data visualization"
 permalink: /visualization/
 ---
 
-This page is under construction.
+## Overview
+
+* You will learn accessibility guidelines for biological visualizations.
+* These guidelines are based on Chartability ([Elavsky et al.](https://www.frank.computer/chartability/)).
+* (Activity) You will evaluate two biological data visualizations using the guidelines.
+
+## [Chartability](https://chartability.fizz.studio/)	
```

Comment:
Seems like there is some division on links as headers ([reddit discussion](https://www.reddit.com/r/accessibility/comments/m8cugz/can_headings_be_links/)). Perhaps we should refrain from that?

Reply:
This is good to know, thanks for sharing this!